spec/command_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+module Xcake
+  describe Command do
+    it 'displays the current version number with the --version flag' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.